artichoke-backend/src/extn/core/time/trampoline.rs

```diff
@@ -472,10 +467,30 @@ pub fn subsec(interp: &mut Artichoke, time: Value) -> Result<Value, Error> {
 
 // Time format
 
-pub fn strftime(interp: &mut Artichoke, time: Value, format: Value) -> Result<Value, Error> {
-    let _ = interp;
-    let _ = time;
-    let _ = format;
-    // Requires a parser.
-    Err(NotImplementedError::new().into())
+pub fn strftime(interp: &mut Artichoke, mut time: Value, format: Value) -> Result<Value, Error> {
```

Comment:
Do you think we can add `asctime` and `ctime` in the same PR using the route through strftime approach that `to_s` does?

Reply:
I had a feeling I was missing some, yes will add!